src/olympia/shelves/validators.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+def validate_criteria(value):
+    url = "https://addons.mozilla.org/api/v4/addons/{}"
```

Comment:
as discussed in the meeting, we don't want this as a constant inside the method.  If we're running against the current server we can do `drf_reverse('v4:addon-search')` (`from rest_framework.reverse import reverse as drf_reverse`) to get the addon search api endpoint.  Note this would mean a change in the criteria because you wouldn't need the "search/" part of it.

Not strictly necessary here, but to get an absolute url, append `settings.EXTERNAL_SITE_URL` to the start of the url.